packages/webpack/loader.js

```diff
@@ -3,9 +3,10 @@
 const { transform } = require("@modular-css/css-to-js");
 
 const DEFAULTS = {
-    styleExport   : true,
-    namedExports  : true,
-    defaultExport : true,
+    styleExport         : true,
+    namedExports        : true,
+    defaultExport       : true,
+    variableDeclaration : "const",
```

Comment:
If `css-to-js` already has a default value for `variableDeclaration` why does it need to be set again here?

Reply:
gone